index.html

```diff
@@ -0,0 +1,1783 @@
+                                <li class="list-inline-item border-end p-1 pe-2"><a class="text-decoration-none text-secondary" href="#">Romania</a></li>
+                            </ul>
+                        </div>
+                        <div class="d-flex">
+                            <h6 class="me-2 text-secondary ">About Us</h6>
+                            
+                            <ul class="list-unstyled list-inline ">
+                                <li class="list-inline-item border-end pe-1 "><a class="text-decoration-none text-secondary" href="#">Announcements</a></li>
+                                <li class="list-inline-item border-end pe-1"><a class="text-decoration-none text-secondary" href="#">Archive</a></li>
+                                <li class="list-inline-item border-end pe-1"><a class="text-decoration-none text-secondary" href="#">Careers</a></li>
+                                <li class="list-inline-item border-end pe-1"><a class="text-decoration-none text-secondary" href="#">Corrections</a></li>
+                                <li class="list-inline-item border-end pe-1"><a class="text-decoration-none text-secondary" href="#">Contact Us</a></li>
+                                <li class="list-inline-item border-end pe-1"><a class="text-decoration-none text-secondary" href="#">Editorial Guidelines</a></li>
+                                <li class="list-inline-item border-end pe-1"><a class="text-decoration-none text-secondary" href="#">Mission Statement</a></li>
+                                <li class="list-inline-item border-end pe-1" ><a class="text-decoration-none text-secondary" href="#">Advertise</a></li>
+                                <li class="list-inline-item border-end pe-1"><a class="text-decoration-none text-secondary" href="#">Copyright</a></li>
+                                <li class="list-inline-item border-end pe-1"><a class="text-decoration-none text-secondary" href="#">Terms & Conditions</a></li>
+                                <li class="list-inline-item border-end pe-1"><a class="text-decoration-none text-secondary" href="#">Privacy Policy</a></li>
+                            </ul>
+                        </div>
+                        <div class="d-flex">
+                            <h6 class="me-3 text-secondary">Cookie Policy</h6>
+                            <ul class="list-unstyled list-inline ">
+                                <li class="list-inline-item border-end pe-1"><a class="text-decoration-none text-secondary" href="#">Terms of Sale</a></li>
+                                <li class="list-inline-item border-end pe-1"><a class="text-decoration-none text-secondary" href="#">Leadership</a></li>
+                                <li class="list-inline-item border-end pe-1"><a class="text-decoration-none text-secondary" href="#">CEO Dev Pragad</a></li>
```

Comment:
- [ ] Please wrap all sections of the footer in a footer tag, it's important to use semantic HTML5 tags.